test/unit/lib/is-array.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
```

Comment:
We don't have a `toString` version, just `typeof x === 'string'`, so I think that's okay.